File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameChallengeQueryForm.kt
package com.sdkwork.backend

data class PlusGameChallengeQueryForm(
    val id: Int? = null,
    val createdAtFrom: String? = null,
    val createdAtTo: String? = null,
    val challengerUserId: Int? = null,
    val targetUserId: Int? = null,
    val arenaId: Int? = null,
    val winnerUserId: Int? = null,
    val gameType: String? = null,
    val status: String? = null,
    val sourceType: String? = null
)

File: sdkwork-backend-sdk-typescript/src/types/plus-game-challenge-query-form.ts
/** 棋牌游戏挑战赛查询表单 */
export interface PlusGameChallengeQueryForm {
  id?: number;
  createdAtFrom?: string;
  createdAtTo?: string;
  /** 挑战方用户ID */
  challengerUserId?: number;
  /** 被挑战方用户ID */
  targetUserId?: number;
  /** 擂台ID */
  arenaId?: number;
  /** 获胜方用户ID */
  winnerUserId?: number;
  /** 玩法类型 */
  gameType?: 'INTERNATIONAL_CHESS' | 'CHINESE_CHESS' | 'GOMOKU' | 'JUNQI' | 'LANDLORD' | 'SHENG_JI' | 'TUO_LA_JI' | 'TEXAS_HOLDEM' | 'MAHJONG' | 'CUSTOM';
  /** 挑战状态 */
  status?: 'PENDING' | 'ACCEPTED' | 'REJECTED' | 'CANCELED' | 'EXPIRED' | 'SETTLED';
  /** 挑战来源 */
  sourceType?: 'SEARCH' | 'LEADERBOARD' | 'ARENA' | 'MANUAL';
}
